web/src/copy_and_paste.ts

```diff
@@ -464,16 +501,19 @@ export function paste_handler_converter(paste_html) {
             return (
                 options.codeBlockStyle === "fenced" &&
                 node.nodeName === "PRE" &&
-                [...node.childNodes].filter((child) => child.textContent.trim() !== "").length ===
+                [...node.childNodes].filter((child) => child.textContent?.trim() !== "").length ===
                     1 &&
-                [...node.childNodes].find((child) => child.textContent.trim() !== "").nodeName ===
+                [...node.childNodes].find((child) => child.textContent?.trim() !== "")?.nodeName ===
                     "CODE"
             );
         },
 
         replacement(_content, node, options) {
             const codeElement = [...node.childNodes].find((child) => child.nodeName === "CODE");
-            const code = codeElement.textContent;
+            const code = codeElement?.textContent ?? "";
```

Comment:
Didn't understand the sense behind continuing on the code path with empty `code` field.